Core/LocalAdmin.cs

```diff
@@ -701,25 +809,15 @@ async void ReaderTaskMethod()
                 if (string.IsNullOrWhiteSpace(input))
                     continue;
 
-                var currentLineCursor = NoSetCursor ? 0 : Console.CursorTop;
-
-                if (currentLineCursor > 0)
-                {
-                    Console.SetCursorPosition(0, currentLineCursor - 1);
-
-                    ConsoleUtil.WriteLine($"{string.Empty.PadLeft(Console.WindowWidth)}>>> {input}", ConsoleColor.DarkMagenta, -1);
-                    Console.SetCursorPosition(0, currentLineCursor);
-                }
-                else
-                    ConsoleUtil.WriteLine($">>> {input}", ConsoleColor.DarkMagenta, -1);
+                ConsoleUtil.WriteLine($">>> {input}", ConsoleColor.DarkMagenta, -1);
 
                 if (!_processRefreshFail && _gameProcess != null)
                 {
                     try
                     {
                         if (_gameProcess.HasExited)
                         {
-                            ConsoleUtil.WriteLine("Failed to send command - the game process was terminated...", ConsoleColor.Red);
+                            ConsoleUtil.WriteLine("Failed to send command - the game process was terminated...", ConsoleColor.Red, inputIntro: false);
```

Comment:
You don't really need the `inputIntro` as far as I remember. I'm not gonna make the exact same comment to every single instance of this specific comment.

Reply:
Interesting. I've never seen that behaviour.